inception_reports/generate_reports_manager.py

```diff
@@ -391,34 +766,161 @@ def export_data(project_data, output_directory=None):
         project_data (dict): The data to be exported.
     """
     current_date = datetime.now().strftime("%Y_%m_%d")
-    directory_name = f'exported_data_{current_date}'
+    directory_name = f"exported_data_{current_date}"
 
     if output_directory is None:
         output_directory = os.path.join(os.getcwd(), directory_name)
     else:
         output_directory = os.path.join(output_directory, directory_name)
-    
+
     if not os.path.exists(output_directory):
         os.makedirs(output_directory)
 
     project_name = project_data["project_name"]
-    with open(f"{output_directory}/{project_name.split('.')[0]}_data_{current_date}.json", "w") as output_file:
+    with open(
+        f"{output_directory}/{project_name.split('.')[0]}_data_{current_date}.json", "w"
```

Comment:
If the project_name comes from the INCEpTION export, I believe the it can contain spaces and other characters you don't want to have in file names. The `slug` is probably the better choice here.